src/SmartComponents/Rules/_rules.scss

```diff
@@ -1,3 +1,3 @@
 .page__rules {
-    // Filler
+    .rulesHideHitsGroup { margin-left: auto; }
```

Comment:
I think we should be sticking to pf styles as much as possible, `pf-u-justify-content-space-between` removed on line https://github.com/RedHatInsights/insights-advisor-frontend/pull/205/files#diff-e0d9c4570235d8fddea19d06db87d149L48 achieved this

Reply:
I took this approach looking at the pf4 simple toolbar sample markup [1].  The original `pf-u-justify-content-space-between` worked perfectly when there were two distinct sets to work with (inside of `Filters` and 1 child passed in), but it's not as clean when passing in multiple children and treating them differently).

[1] https://pf4.patternfly.org/components/Toolbar/examples-full/?component=Toolbar%20simple